remote/remote_state_s3.go

```diff
@@ -435,9 +420,19 @@ func CreateS3BucketWithVersioningSSEncryptionAndAccessLogging(s3Client *s3.S3, c
 	}
 
 	if config.SkipBucketAccessLogging {
-		terragruntOptions.Logger.Printf("Access Logging is disabled for the remote state AWS S3 bucket %s using 'skip_bucket_accesslogging' config.", config.remoteStateConfigS3.Bucket)
-	} else if err := EnableAccessLoggingForS3BucketWide(s3Client, &config.remoteStateConfigS3, terragruntOptions); err != nil {
-		return err
+		terragruntOptions.Logger.Printf("___WARNING___: The terragrunt configuration option 'skip_bucket_accesslogging' is now deprecated. To enable access logging for bucket %s, please provide property `accesslogging_bucket_name` in the terragrunt config file. For more details, please refer to the Terragrunt documentation.", config.remoteStateConfigS3.Bucket)
+	}
+
+	if config.AccessLoggingBucketName != "" {
```

Comment:
There should be a log message here indicating that terragrunt is going to create the access logging bucket and enable it.

Reply:
did you mean something like in the method being called (See lines 654-656). I think it's a fair point though, as it will read easier if the logging as all done at the same level.

Reply:
No I mean that since this is doing multiple steps, it would help to have a log message here indicating the two steps it is doing (creating access logging bucket and then hooking it up). I think that is slightly better in terms of debuggability.

It's not a big deal though so if you disagree that is helpful, feel free to ignore (hence labeled as `nit`).

Reply:
ah I see! that's a good suggestion indeed! will get it in there :)